### PR TITLE
Add dataset and instructions.

### PR DESCRIPTION
**Pull request by willardmr**
Add dataset and instructions for the ML Engineer case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+## Purpose
+
+As the next step in the interview process for the ML Engineer job at Reel, we will ask you to complete this project. We will use your solution to the project to get a better understanding of your technical abilities and how you approach a technical challenge. Your solution will also be the starting point for our subsequent technical interview. The project is intended to be a small representation of the kind of work we do at Reel.
+
+## Problem description
+
+As a Balancing Responsible Party in the Danish markets we are responsible for placing orders into the day-ahead electricity market auction.  An order includes a volume, price, and an hourly timestamp.  If executed these orders are converted in trades and we buy or sell the given volumes at the settled price.
+
+Due to operational requirements we place orders each week for the next 7 days.  Our data is delayed by 3 days so we need to forecast 10 days into the future.  We need accurate forecasts so we do not buy or sell electricity unnecessarily.
```
Comment (RagingSeabass) [inline README.md]:
```suggestion
Due to operational requirements we place orders each week for the next 7 days.  Our input data is delayed by 3 days so we need to forecast 10 days into the future.  We need accurate forecasts so we do not buy or sell electricity unnecessarily.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,46 @@
...
+
+We expect the project to take 2-3 hours to complete. You are always more than welcome to reach out to us if something needs clarification.
+
+## Your task
+
+#### 1. Train a model to forecast our consumption
+
+We have about two and a half years of hourly consumption data in the dataset.csv file.  Using this dataset, create a model that can forecast 10 days into the future.
```
Comment (RagingSeabass) [inline README.md]:
```suggestion
The dataset.csv file contains about two and a half years of hourly consumption data.  Using this dataset, create a model that can forecast 10 days into the future.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
...
+## Submitting your work
+
+1. Clone this repository locally
+2. Create and switch to a local branch for development
+3. Develop your solution
+4. Once you're done and have committed everything, create a patch of your changes
+   - `git diff main > <your_name>.patch`
+5. Send the following to christian@reel.energy
+   1. The patch file
+   2. The description of your solution in pdf format
```
Comment (RagingSeabass) [inline README.md]: Patch files turned out to be difficult to work with, so I suggest we simply use zips.

```suggestion
1. Clone this repository locally
2. Develop your solution
3. Once you're done, create zip of the main folder and name it `<your_name>.zip`
4. Send the following to christian@reel.energy
   1. The zip file
   2. The description of your solution in pdf or md format
```

File: README.md (hunk 2 above)
Comment (RagingSeabass) [inline README.md:19]: Is it a single MP? Maybe worth specifying.

File: README.md [hunk 4]
```diff
@@ -1 +1,46 @@
...
+
+## How we evaluate
+
+We focus on three main points in our evaluation:
+
+1. How you solved the task. There are many ways to approach a task like this.
+2. Your reasoning about choosing this solution.
+3. How you would manage a model deployed to production over time.
```
Comment (RagingSeabass) [inline README.md]:
```suggestion
3. How you would manage a model deployed to production over time - we will discuss this at the interview.
```

File: README.md (hunk 2 above)
Comment (willardmr) [reply README.md]: I figured that is more detail than needed for the problem.  And I want to give some hypotheticals like "If this is our entire consumption in a bidding zone and we need to add companies, how would you handle that?"

Comment (RagingSeabass) [reply README.md]: Makes sense.

**Review by RagingSeabass — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,44 @@
...
+
+#### 2. Evaluate your model
+
+Perform analysis of your trained model's predictions.
+
+#### 3. Explain your solution
+
+Write a short (1200 characters max) explanation of how your solution works, why you chose that particular solution, and justify your explanations using the data.
+
+## Submitting your work
```
Comment (drobnyd) [inline README.md:29]: Not sure if we want to specify this but we would probably expect them to hand in a python or Jupyter notebook project?

Comment (willardmr) [reply README.md:29]: @RagingSeabass what do you think?  I guess I am fine with them handing in whatever (R, Python, Latex, Pdf, other?), but maybe we should require something more specific?

File: README.md (hunk 2 above)
Comment (drobnyd) [reply README.md]: Just flagging that this discussion is public

Comment (drobnyd) [reply README.md]: (so maybe we want to delete the comment revealing one of the interview questions @willardmr)

Comment (willardmr) [reply README.md]: Ok sure

File: README.md (hunk 5 above)
Comment (willardmr) [reply README.md:29]: Yeah we prefer Python, but I guess we will see what we get.

**Pull request merged**